test/etl_test.py

```diff
@@ -208,13 +208,13 @@ def test_extract_eia860(self, pudl_datastore_fixture):
         """Spot check extraction of eia860 excel files."""
         extractor = pudl.extract.eia860.Extractor(pudl_datastore_fixture)
         assert "Ownership" in extractor.load_excel_file(
-            2018, "ownership").sheet_names
+            part=2018, page="ownership").sheet_names
 
     def test_extract_eia923(self, pudl_datastore_fixture):
         """Spot check extraction eia923 excel files."""
         extractor = pudl.extract.eia923.Extractor(pudl_datastore_fixture)
         assert "Page 3 Boiler Fuel Data" in extractor.load_excel_file(
-            2018, "stocks").sheet_names
+            part=2018, page="stocks").sheet_names
```

Comment:
this is another reason why using `**kwargs` might be cleaner here. Instead of using vague `part=$somevalue`, using year=2018, or `year_month="2018-08"` makes it more apparent of what we are passing through.

Reply:
I very much like the idea of not needing to specify exactly what all of the partitions are, since the possibilities will probably change over time (e.g. epacems has year + state, ferceqr has year + quarter, the LMP data may be partitioned by which grid operator it came from, etc.) and doing the right thing can be left to the extract function, which is going to be dataset specific. But this will affect the metadata that we use to specify filename / column / tab maps for the spreadsheets.